x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/rest/datafeeds/RestStartDatafeedAction.java

```diff
@@ -71,12 +71,14 @@ protected RestChannelConsumer prepareRequest(RestRequest restRequest, NodeClient
         }
         return channel -> {
             client.execute(StartDatafeedAction.INSTANCE, jobDatafeedRequest,
-                    new RestBuilderListener<AcknowledgedResponse>(channel) {
+                    new RestBuilderListener<NodeAcknowledgedResponse>(channel) {
 
                         @Override
-                        public RestResponse buildResponse(AcknowledgedResponse r, XContentBuilder builder) throws Exception {
+                        public RestResponse buildResponse(NodeAcknowledgedResponse r, XContentBuilder builder) throws Exception {
+                            // This doesn't use the toXContent of the response object because we rename "acknowledged" to "opened"
```

Comment:
```suggestion
                            // This doesn't use the toXContent of the response object because we rename "acknowledged" to "started"
```